realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -491,71 +477,47 @@ public void addAt(long rowIndex, Object... values) {
             Object value = values[(int)columnIndex];
             switch (colTypes[(int)columnIndex]) {
             case BOOLEAN:
-                nativeInsertBoolean(nativePtr, columnIndex, rowIndex, (Boolean)value);
+                setBoolean(columnIndex, rowIndex, (Boolean)value);
```

Comment:
@bmunkholm All the `insert_xxx` have been removed from core. Actually I'd like to remove this `addAt` as well since it is not used in the real code. But lots of our JNI testing is relying on it... So make it `protected` and keep it there for only internal testing usage until at some point we have a real batch adding function.